README.md

```diff
@@ -303,19 +311,29 @@ started from the project's virtual environment in `.venv`:
 python scripts/envhook.py install
 ```
 
+The script works by adding a `sitecustomize.py` file to your virtual 
+environment. If a different `sitecustomize` module is already present in your 
+Python path, its `sitecustomize.py` file must be renamed or removed before the 
```

Comment:
@jessebrennan I thought you implemented a check for that?

Reply:
There is a check. The command warns you if there is already a sitecustomize. These instructions are still helpful though, if you lack the context of what sitecustomize is, or are unsure how to find the file.

Reply:
OK